packages/cozy-pouch-link/src/CozyPouchLink.js

```diff
@@ -139,7 +139,7 @@ class PouchLink extends CozyLink {
         PouchDB.plugin(plugin)
       }
       const doctypes = getPersistedSyncedDoctypes()
-      for (const doctype of doctypes) {
+      for (const doctype of Object.keys(doctypes)) {
```

Comment:
Antoher (purely cosmetic) way to write this may be `for (const doctype in doctypes)`

`of` iterates over elements while `in` iterates over indexes